src/main/java/com/devqoo/backend/auth/jwt/JwtProvider.java

```diff
@@ -0,0 +1,134 @@
+package com.devqoo.backend.auth.jwt;
+
+import com.devqoo.backend.auth.security.CustomUserDetails;
+import com.devqoo.backend.common.exception.BusinessException;
+import com.devqoo.backend.common.exception.ErrorCode;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.MalformedJwtException;
+import io.jsonwebtoken.UnsupportedJwtException;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import java.util.List;
+import javax.crypto.SecretKey;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtProvider {
+
+    private final SecretKey accessKey;
+    private final SecretKey refreshKey;
+
+    @Value("${JWT_ACCESS_TIME}")
+    private int ACCESS_EXPIRE_TIME;
+    @Value("${JWT_REFRESH_TIME}")
+    private int REFRESH_EXPIRE_TIME;
```

Comment:
대문자 + 언더바 = 상수
```suggestion
    @Value("${JWT_ACCESS_TIME}")
    private static final int ACCESS_EXPIRE_TIME;
    @Value("${JWT_REFRESH_TIME}")
    private static final int REFRESH_EXPIRE_TIME;
```

근데 주입받으니까 static을 사용하지 못할 것 같아요 
`accessExpireTime` 으로 수정 가능

굳이 대문자 + 언더바를 유지할 필요는 없을 것 같아요. 다른 방법도 찾아봐도 좋고요!

Reply:
JwtProvider의 생성자 부분에서 추가 하도록 하겠습니다.